### ISSUE TITLE
output length of tandem repeats

### ISSUE THREAD

**Issue by jorondo1** [Benson-Genomics-Lab/TRF#14]
Hi, I am trying to figure out how to estimate the total length of tandem repeats found in a sequence and I can't figure out which metric to use. I have several genomes I would like to analyse, I would be interested in how to generate this data from TRF output, do you have any recommendation ?

**Comment by bricoletc** [comment]
(Not a maintainer of this project but) what exactly do you mean by 'total length'? Sum of `Pattern size * copy-number` for all repeats identified in a sequence? Or all locations in a sequence covered by at least one tandem repeat?

**Comment by jorondo1** [comment]
I want to find out what % of the genome is tandem repeats. Would `Pattern size * copy-number / genome length` be the way to go?

**Comment by bricoletc** [comment]
Not quite- you'd need to account for positionally-overlapping repeats. I guess ideal would be a BED output describing locations of all the tandem repeats, followed by `bedtools genomecov` for example. 

Bed output already requested in #12. I'd actually be happy to propose an implementation, but it seems like maintainers are not responsive overall - @yzhernand @GaryEBenson ?

**Comment by tomsauv** [comment]
Hi,
This is couple years old now but In case this is still of any help, you may use the following script to get total length of detected tandems. 

https://github.com/tomsauv/TROP-Tandem-Repeat-Overlap-Parser

The script merge the coordinates of overlapping tandem to get total length.

Hope this helps
